images/openshift-ci-azure/azure-cli.repo

```diff
@@ -0,0 +1,6 @@
+[azure-cli]
+name=Azure CLI
+baseurl=https://packages.microsoft.com/yumrepos/azure-cli
+enabled=1
+gpgcheck=1
+gpgkey=https://packages.microsoft.com/keys/microsoft.asc
```

Comment:
I'd favour importing this gpg key in our repo. If we import the gpg key from the same location where we download the packages, it does not add a lot of trust.

Reply:
@cfergeau we are using same https://github.com/code-ready/crc/blob/main/images/openshift-ci/google-cloud-sdk.repo#L7-L8 for gcp but may be a separate PR to do it for both place?